### ISSUE TITLE
simulate_with_bioscrape_via_sbml: "NameError: name 'bioscrape' is not defined"

### ISSUE THREAD

**Issue by mkapasiawala** [BuildACell/BioCRNPyler#114]
In chemical_reaction_network.py, the function `simulate_with_bioscrape_via_sbml` does not contain the line 
from bioscrape.simulator import py_simulate_model
so an error is given in trying to simulate a CRN using this. I think there should be an import either before any classes are defined or in the `simulate_with_bioscrape_via_sbml` function itself, as was done for `simulate_with_bioscrape`.

**Comment by zoltuz** [comment]
Hi,
bioscrape is independent of BioCRNPlyer, please make sure you have followed the installation steps: [how to install bioscrape](https://github.com/biocircuits/bioscrape/wiki/Installation)

An alternative simulation tool is RoadRunner, the installation is explained here: https://sys-bio.github.io/roadrunner/python_docs/installing_rr.html

**Comment by mkapasiawala** [comment]
Hi, sorry if my issue was not properly described. I am aware that bioscrape and biocrnpyler are two different things, and I already have bioscrape installed. But the biocrnpyler function `simulate_with_bioscrape_via_sbml` needs to import the bioscrape package to simulate a CRN with bioscrape. The biocrnpyler functions `runsim_roadrunner` and `simulate_with_bioscrape` import the packages `roadrunner` and `bioscrape`, respectively, so that the packages can be used for simulating the CRNs; is it not true that `simulate_with_bioscrape_via_sbml` needs a similar import to simulate a CRN with bioscrape? 

I thought this was the error because when I tried simulating a CRN with `simulate_with_bioscrape_via_sbml`, I got the error, but not when I tried `simulate_with_bioscrape`, but maybe there is another issue.

**Comment by ayush9pandey** [comment]
Yeah, I think the import statement should be there. Although I remember using this function earlier and it worked :O ?

Anyways, I think for now we should put all of the bioscrape import statements in try except blocks.

**Comment by zoltuz** [comment]
I got it now! I patched it in #116, it will get merged to dev after review.

@mkapasiawala Thanks for reporting this! It will take a while these changes get merged into the master branch.

Until then, you can install the dev branch:
```
pip uninstall biocrnpyler
pip install git+git://github.com/BuildACell/BioCRNPyler.git@dev
```
An example from google colab:
<img width="658" alt="84891386-b2858f80-b093-11ea-8517-9fc2771669ca" src="https://user-images.githubusercontent.com/17138838/86163163-c4d9e180-bb07-11ea-9990-e913ba8a9c49.png">

**Comment by zoltuz** [comment]
PR #116 has solves this issue. it's available on the dev branch. @mkapasiawala Thanks for reporting this issue!
